higherKindCore/src/main/scala/tofu/higherKind/bi/FunctorB.scala

```diff
@@ -0,0 +1,9 @@
+package tofu.higherKind.bi
+
+trait FunctorB[U[f[_, _]]] {
```

Comment:
We should think of better naming for bi-analogs of *K typeclasses.

Reply:
Ok, but what

Reply:
Agree with @catostrophe, *K naming convention has to be preserved. `FunctorBK` or `FunctorBiK` looks ok.

Reply:
@oskin1 Why? What is so special about K?